gems/pending/spec/spec_helper.rb

```diff
@@ -32,3 +36,30 @@
   config.backtrace_exclusion_patterns << %r{/lib\d*/ruby/[0-9]}
   config.backtrace_exclusion_patterns << %r{/gems/[0-9][^/]+/gems/}
 end
+
+#
+# So tests can clear class-level caches between examples.
+# TODO: Add this to the azure-armrest gem.
+#
+class Azure::Armrest::ArmrestService
+  def self.clear_caches
+    @@providers_hash = {}
+    @@tokens         = {}
+    @@subscriptions  = {}
+  end
+end
+
+VCR.configure do |c|
+  c.cassette_library_dir = TestEnvHelper.recordings_dir
+  c.hook_into :webmock
+
+  c.allow_http_connections_when_no_cassette = false
+  c.default_cassette_options = {
+    :record                         => :once,
+    :allow_unused_http_interactions => true
+  }
+
+  TestEnvHelper.vcr_filter(c)
+
+  # c.debug_logger = File.open(Rails.root.join("log", "vcr_debug.log"), "w")
```

Comment:
I agree, the directory is wrong for gems/pending... maybe this should along with debug logging followup PR so we can get this one in?  What do you think?